src/components/display/Dropdown.jsx

```diff
@@ -308,10 +324,19 @@ const Dropdown = React.forwardRef(function DropdownFn(
 
 	const clickOutsidePopper = useCallback(
 		(e) => {
-			dropdownRef.current &&
+			if (
+				dropdownRef.current &&
 				e.target !== dropdownRef.current &&
 				!dropdownRef.current.contains(e.target) &&
+				// check if the attribute is in the event path
+				!reduce(
```

Comment:
Maybe `find` or `some` are better since they stop when a "true" is found

Reply:
I'm kind of ashamed of myself right now, fixing